src/User32.Desktop/User32+KEYBDINPUT.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="KEYBDINPUT"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// Contains information about a simulated keyboard event.
+        /// </summary>
+        [StructLayout(LayoutKind.Sequential)]
+        [OfferIntPtrPropertyAccessors]
```

Comment:
Please verify that this attribute is working for you.
At the moment, you've left out the attribute on the Compile item in csproj that would cause this attribute to actually do anything.
Same for the other file(s) you've added this attribute to.